sdcm/utils/argus.py

```diff
@@ -8,7 +9,17 @@
 LOGGER = logging.getLogger(__name__)
 
 
-def get_argus_client(run_id: UUID | str) -> ArgusSCTClient:
+def is_uuid(uuid) -> bool:
+    try:
+        UUID(uuid)
+        return True
+    except (ValueError, AttributeError):
+        return False
+
+
+def get_argus_client(run_id: UUID | str) -> ArgusSCTClient | MagicMock:
+    if not run_id or not is_uuid(run_id):
+        return MagicMock()
```

Comment:
In general the return a mock approach can be tricky, since if there a code looking at some fields, or code expecting some structured response, it might fail when the mock is used. (we've seen such cases with LWT validation)

option to disable Argus should be a first class feature

Reply:
Fixed. I think if we want disabling to be a first class features, then the client itself should have a `dry_run` or `disabled` init parameter, which just makes every command instantly return (as they produce no output currently, this should be easy).

Reply:
Please open an issue for Argus/SCT for this clearer disablement

Reply:
Done, scylladb/argus#217